src/Altinn.Notifications.Core/Models/ContactPoints/OrganizationContactPoints.cs

```diff
@@ -19,4 +26,30 @@ public class OrganizationContactPoints
     /// Gets or sets a list of official email addresses
     /// </summary>
     public List<string> EmailList { get; set; } = [];
+
+    /// <summary>
+    /// Gets or sets a list of user registered contact points associated with the organization.
+    /// </summary>
+    public List<UserContactPoints> UserContactPoints { get; set; } = [];
+
+    /// <summary>
+    /// Create a new instance with the same values as the existing instance
+    /// </summary>
+    /// <returns>The new instance with copied values.</returns>
+    public OrganizationContactPoints CloneWithoutUsers()
+    {
```

Comment:
wondering if we could clone without any contactPoints, not just excluding user contact points. As it is intended used the email and mobile number list won't be populated either way. Cannot see any use cases where they might be for the time being either

Reply:
I changed it.